weiss_wsg32_support/package.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<package>
+  <name>weiss_wsg32_support</name>
+  <version>0.0.1</version>
+  <description>The weiss/schunk wsg32 gripper support package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag --> 
+  <maintainer email="rfa-fr@ipa.fraunhofer.de">Florian Roeser</maintainer>
```

Comment:
Looks good,
unfortunately Florian is not with us anymore. Would you please replace Florian with myself as being the (second) maintainer:
<maintainer email="lorenz.halt@ipa.fraunhofer.de">Lorenz Halt</maintainer>

Reply:
Done